libgalois/include/galois/runtime/config.h.in

```diff
@@ -1,6 +1,8 @@
 #ifndef GALOIS_RUNTIME_CONFIG_H
 #define GALOIS_RUNTIME_CONFIG_H
 
-#cmakedefine GALOIS_USE_LONGJMP_ABORT
+#if !(defined(GALOIS_USE_LONGJMP_ABORT) || defined(GALOIS_USE_EXCEPTION_ABORT))
```

Comment:
Yah, I was thinking of leaving the configure_file machinery in place since we can use that for other things later (e.g. setting the Galois version info up at the CMake level and then having it automatically set up the headers and source files with the correct version info).